docs/repo/property-pages/visibility-conditions.md

```diff
@@ -38,16 +49,17 @@ By way of example, the simplest expression is a literal value.
 
 To make this language useful, we need the ability to perform operations with these values. Here are some examples:
 
-| Expression             | Value   |
-|------------------------|---------|
-| `(not true)`           | `false` |
-| `(or true false)`      | `true`  |
-| `(eq 1 2)`             | `false` |
-| `(lt 1 2)`             | `true`  |
-| `(add 1 2)`            | `3`     |
-| `(add 1 2 3)`          | `6`     |
-| `(concat "a" "b")`     | `"ab"`  |
-| `(concat "a" "b" "c")` | `"abc"` |
+| Expression             | Value             |
+|------------------------|-------------------|
+| `(not true)`           | `false`           |
+| `(or true false)`      | `true`            |
+| `(eq 1 2)`             | `false`           |
+| `(lt 1 2)`             | `true`            |
+| `(add 1 2)`            | `3`               |
+| `(add 1 2 3)`          | `6`               |
+| `(concat "a" "b")`     | `"ab"`            |
+| `(concat "a" "b" "c")` | `"abc"`           |
+| `(list "a" "b" "c")`   | `["a", "b", "c"]` |
```

Comment:
I'm curious what `list` is used for. I didn't see a usage. Given lists are such a common construct, we could add language support for them if needed (i.e. exactly what's written on the right column).